CMSgov/managed-care-review
# MR-2955: Display CMS users in a table

## Summary

Added a read-only table of CMS users at the route `/settings, visible only to admins.  Eventually, we'll want this table to be the editable source of truth for information about our users, including their email addresses and division assignments.

Also added Iroh, our first local admin user.

@mojotalantikite Do we want a uniqueness constraint on emails in the User table?  In looking at the table after our tests run, I see lots of users with the same email.

#### Screenshots

<img width="1271" alt="image" src="https://user-images.githubusercontent.com/8964335/223177430-b64a1277-3284-4222-b78d-10db62adc6da.png">

#### Test cases covered

Added a test for the table and contents at `/settings`, as well as some to cover the new admin user.

## QA guidance

If you log in as Iroh, locally and navigate to `/settings`, you should see a table of CMS users.
